### PR TITLE
feat(jobs/pingcap/tidb): enable condition triggering on trunk branches

### PR DESCRIPTION
**Pull request by wuhuizuo**
Branches:
- trunk branch
- feature branches
- latest release branch

Depended on: https://github.com/ti-community-infra/configs/pull/812

### CODE REVIEW

File: prow-jobs/pingcap-tidb-latest-presubmits.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ presubmits:
     - name: pingcap/tidb/ghpr_unit_test
       agent: jenkins
       decorate: false # need add this.
-      always_run: true
+      skip_if_only_changed: "(\\.md|Dockerfile|OWNERS|OWNERS_ALIASES)$"
```
Comment (purelind) [inline prow-jobs/pingcap-tidb-latest-presubmits.yaml:47]: The current pipelines will automatically trigger after a new commit push. Will this behavior change after adjustment?

Comment (wuhuizuo) [reply prow-jobs/pingcap-tidb-latest-presubmits.yaml:47]: no, it judge the the diff of the whole pull request, not the commit changed scope.

Comment (purelind) [reply prow-jobs/pingcap-tidb-latest-presubmits.yaml:47]: My concern is whether the pipeline can be automatically triggered based on the new commit after modifying the code and pushing it.

**Pull request merged**